XMonad/Layout/Magnifier.hs

```diff
@@ -119,6 +124,17 @@ magnify
 magnify cz mt start = ModifiedLayout $
     Mag 1 (fromRational cz, fromRational cz) (bool Off On start) mt
 
+magnifyxy
```

Comment:
Maybe add some top-level docs here along the lines of "Like 'magnify', but with the ability to specify different amounts of horizontal and vertical magnification"

Reply:
Is top-level docs like how it is for "magnify"?

As in `-- | Like magnify.....`?